styleguide/derek/_partials/_footerLp.ejs

```diff
@@ -5,52 +5,48 @@
-                <i class="fa fa-2x fa-facebook-official fa-stack-2x"></i>
-              </span>
+            <a href="https://www.facebook.com/rackspace" target="_blank" class="basement-iconListItemLink">
+                <i class="fa fa-2x fa-facebook-official"></i>
             </a>
           </li>
           <li class="basement-iconListItem">
-            <a href="https://twitter.com/Rackspace" target="_blank">
-              <span class="fa-stack">
-                <i class="fa-square fa-stack-1x basement-iconBg"></i>
-                <i class="fa fa-2x fa-twitter-square fa-stack-2x"></i>
-              </span>
+            <a href="https://twitter.com/Rackspace" target="_blank" class="basement-iconListItemLink">
+                <i class="fa fa-2x fa-twitter"></i>
             </a>
           </li>
           <li class="basement-iconListItem">
-            <a href="https://www.youtube.com/rackspace" target="_blank">
-              <span class="fa-stack">
-                <i class="fa-square fa-stack-1x basement-iconBg"></i>
-                <i class="fa fa-2x fa-youtube-square fa-stack-2x"></i>
-              </span>
+            <a href="https://www.youtube.com/rackspace" target="_blank" class="basement-iconListItemLink">
+                <i class="fa fa-2x fa-youtube"></i>
             </a>
           </li>
           <li class="basement-iconListItem">
-            <a href="https://plus.google.com/+Rackspace" target="_blank">
-              <span class="fa-stack">
-                <i class="fa-square fa-stack-1x basement-iconBg"></i>
-                <i class="fa fa-2x fa-google-plus-square fa-stack-2x"></i>
-              </span>
+            <a href="https://plus.google.com/+Rackspace" target="_blank" class="basement-iconListItemLink">
+                <i class="fa fa-2x fa-google-plus"></i>
             </a>
           </li>
           <li class="basement-iconListItem">
-            <a href="https://www.linkedin.com/company/rackspace-hosting/" target="_blank">
-              <span class="fa-stack">
-                <i class="fa-square fa-stack-1x basement-iconBg"></i>
-                <i class="fa fa-2x fa-linkedin-square fa-stack-2x"></i>
-              </span>
+            <a href="https://www.linkedin.com/company/rackspace-hosting/" target="_blank" class="basement-iconListItemLink">
+                <i class="fa fa-2x fa-linkedin"></i>
             </a>
           </li>
         </ul>
       </div>
-      <div class="col-sm-3 col-sm-push-4 col-xs-6">
-        <p class="uppercase copyright basementLp-copyright">&reg; 2016 Rackspace US, Inc.</p>
+    <div class="col-sm-4">
+      <div class="region-select">
+        <span class="uppercase white basement-region-select">Change your region:</span>
+        <a class="basement-flag-link" data-toggle="modal" id="region" data-target="#regionSelect">
+          <div class="basement-flag-icon basement-flag-uk"></div>
+        </a>
+      </div>
+    </div>
```

Comment:
Can you indent this section to match the other column below. It's a little confusing trying to parse it.